src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -2425,5 +2426,27 @@ public Response getSignedUrl(@Context ContainerRequestContext crc, JsonObject ur
         
         return ok(Json.createObjectBuilder().add(ExternalToolHandler.SIGNED_URL, signedUrl));
     }
- 
+
+    /**
+     * For testing only. Download a file from the file system.
+     */
+    @GET
+    @AuthRequired
+    @Path("/localfile")
+    public Response getLocalFile(@Context ContainerRequestContext crc, @QueryParam("pathToFile") String pathToFile) {
+        try {
+            AuthenticatedUser user = getRequestAuthenticatedUserOrDie(crc);
+            if (!user.isSuperuser()) {
+                return error(Response.Status.FORBIDDEN, "Superusers only.");
+            }
+        } catch (WrappedResponse wr) {
+            return wr.getResponse();
+        }
+        try {
+            return ok(new FileInputStream(pathToFile));
```

Comment:
even as a protected admin call, I'd suggest limiting this to some subdirectory

Reply:
Hmm, yeah, I can easily download `/etc/passwd` which seems pretty suboptimal. 😬

Reply:
In fa6f850 I limited the API to downloading files from `/tmp`.